server/src/test/java/org/eclipse/lsp/cobol/core/model/tree/NodeProcessingTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+import com.google.common.collect.ImmutableList;
+import org.eclipse.lsp.cobol.core.messages.MessageTemplate;
+import org.eclipse.lsp.cobol.core.model.SyntaxError;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/** Test that nodes can be processed in several steps */
+class NodeProcessingTest {
+  private static final SyntaxError ERROR_1 = getError("First error message");
+  private static final SyntaxError ERROR_2 = getError("Second error message");
+  private static final SyntaxError ERROR_3 = getError("Third error message");
+
+  @Test
+  void testSimple() {
+    /** Do nothing */
```

Comment:
Why the javadocs here are below the method names?

Reply:
This javadocs belong to class.

Reply:
Oops, my bad